src/fiskaltrust.Launcher/Commands/Common.cs

```diff
@@ -24,13 +24,17 @@ public CommonCommand(string name) : base(name)
             AddOption(new Option<string?>("--log-folder"));
             AddOption(new Option<LogLevel?>("--log-level"));
             AddOption(new Option<string>("--launcher-configuration-file", getDefaultValue: () => "launcher.configuration.json"));
+            AddOption(new Option<string>("--legacy-config-file", getDefaultValue: () => "fiskaltrust.exe.config"));
```

Comment:
what do you think about the following?
reuse the `launcher-configuration-file` parameter and check if its file extension is `config` or `json`?

Reply:
I'd rather not do that, I think 🤔 see comment below